src/main/java/com/project/service/AdminCommentServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.project.service;
+
+import com.project.exception.AdminCommentNotFoundException;
+import com.project.model.AdminComment;
+import com.project.repository.AdminCommentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class AdminCommentServiceImpl implements AdminCommentService {
+
+    private final AdminCommentRepository commentRepository;
+
+    @Autowired
+    public AdminCommentServiceImpl(AdminCommentRepository commentRepository) {
+        this.commentRepository = commentRepository;
+    }
+
+    @Override
+    public List<AdminComment> getAll() {
+        return new ArrayList<>(commentRepository.findAll());
+    }
+
+    @Override
+    public AdminComment getById(Long id) throws AdminCommentNotFoundException {
+        return commentRepository.findById(id).orElseThrow(() -> new AdminCommentNotFoundException(id));
+    }
+
+    @Override
+    public AdminComment addAdminComment(AdminComment adminComment) {
+        adminComment.setId(null);
```

Comment:
Why do you need this?

Reply:
reworked the `ServiceImpl` classes so that `null` values are no longer assigned